app/assets/javascripts/application.js

```diff
@@ -1,3 +1,4 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
```

Comment:
Hi Tyler!

Thanks for the feedback. I just updated everything on my end and everything
still works as needed. Please let me know if you need me to submit a new
pull request.

In the meantime, I will be embarking on the next task,

Thanks!
Ana

On Mon, Feb 29, 2016 at 1:11 PM, Tyler Bettilyon <notifications@github.com>
wrote:

> In app/assets/javascripts/application.js
> <https://github.com/gSchool/rails-bug-01/pull/9#discussion_r54478355>:
>
> > @@ -1,3 +1,4 @@
> >  //= require jquery
> >  //= require jquery_ujs
> >  //= require_tree .
> > +
>
> Please remove this extra newline -- it's a little nit picky but we don't
> need this file to be changed in the commit history. It will make our lives
> easier later if we have to revert changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gSchool/rails-bug-01/pull/9/files#r54478355>.
>